macros/src/lib.rs

```diff
@@ -299,10 +304,7 @@ enum Piece<'a> {
 
 impl Piece<'_> {
     fn is_str(&self) -> bool {
-        match self {
-            Piece::Str(_) => true,
-            _ => false,
-        }
+        matches!(self, Piece::Str(_))
```

Comment:
the MSRV of `ufmt` 1.0 is 1.34. this macro was added in 1.42 so it cannot be used